manager/Makefile

```diff
@@ -42,6 +42,7 @@ generate: $(TOOLBIN)/controller-gen
 .PHONY: manifests
 manifests: $(TOOLBIN)/controller-gen
 	$(TOOLBIN)/controller-gen $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(TOOLBIN)/controller-gen crd:trivialVersions=true output:crd:artifacts:config=$(ROOT_DIR)/charts/m4d-crd/templates/ paths=./apis/...
```

Comment:
Can you please also remove the `CRD_OPTIONS` above and remove the variable in general?

Reply:
Sure (I didn't touch it because eventually it will be removed altogether but you're right)